graphql/tests/query/spruceConfig/queries/container_pools.graphql

```diff
@@ -0,0 +1,12 @@
+query {
+  spruceConfig {
+    containerPools {
+      pools {
+        id
+        distro
+        maxContainers
+        port
```

Comment:
Is this supposed to include max images? It doesn't matter that much given that we don't use any value but the default, but I was just wondering.

Reply:
I think I might just leave it out for now since `APIContainerPools` doesn't have it! 🤔

Reply:
SGTM! This setting will hopefully go away one day anyways 🙏